### PR TITLE
Added tests for WorkspaceActivityService

### PR DESCRIPTION
**Pull request by mkuznyetsov**

### CODE REVIEW

**Comment by skabashnyuk** [comment]
ок

**Comment by akorneta** [comment]
ok

File: core/codenvy-hosted-workspace-activity/src/test/java/com/codenvy/activity/server/WorkspaceActivityServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+        Response response = given().when().put(SERVICE_PATH + '/' + WORKSPACE_ID);
+
+        // then
+        assertEquals(response.getStatusCode(), 403);
+    }
+
+    @Test
+    public void shouldNotUpdateWorkspaceActivityOnNotRunningWorkspace() throws NotFoundException, ServerException {
+        // given
+        final WorkspaceImpl workspace = createWorkspace(USER_ID, WorkspaceStatus.STARTING);
```
Comment (garagatyi) [inline core/codenvy-hosted-workspace-activity/src/test/java/com/codenvy/activity/server/WorkspaceActivityServiceTest.java]: Check for other states as well

File: core/codenvy-hosted-workspace-activity/src/test/java/com/codenvy/activity/server/WorkspaceActivityServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+    private WorkspaceActivityService workspaceActivityService;
+
+    @BeforeMethod
+    public void setUp() {
+        workspaceActivityService = new WorkspaceActivityService(workspaceActivityManager, workspaceManager);
+    }
+
+    @Test
+    public void shouldUpdateWorkspaceActivityOnRunningWorkspace() throws NotFoundException, ServerException {
```
Comment (garagatyi) [inline core/codenvy-hosted-workspace-activity/src/test/java/com/codenvy/activity/server/WorkspaceActivityServiceTest.java]: I think `...OfRunningWorkspace` looks better

File: core/codenvy-hosted-workspace-activity/src/test/java/com/codenvy/activity/server/WorkspaceActivityServiceTest.java [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+        // when
+        Response response = given().when().put(SERVICE_PATH + '/' + WORKSPACE_ID);
+
+        // then
+        assertEquals(response.getStatusCode(), 204);
+        verify(workspaceActivityManager).update(eq(WORKSPACE_ID), anyLong());
+    }
+
+    @Test
+    public void shouldThrowExceptionWhenUpdatingWorkspaceActivity() throws NotFoundException, ServerException {
```
Comment (garagatyi) [inline core/codenvy-hosted-workspace-activity/src/test/java/com/codenvy/activity/server/WorkspaceActivityServiceTest.java]: When I see name of this method method I think that it check whether tested class throws exception on any activity or not.
Please rename it to show the real usecase when exception is thrown

File: core/codenvy-hosted-workspace-activity/src/test/java/com/codenvy/activity/server/WorkspaceActivityServiceTest.java [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+
+        // when
+        Response response = given().when().put(SERVICE_PATH + '/' + WORKSPACE_ID);
+
+        // then
+        assertEquals(response.getStatusCode(), 403);
+    }
+
+    @Test
+    public void shouldNotUpdateWorkspaceActivityOnNotRunningWorkspace() throws NotFoundException, ServerException {
```
Comment (garagatyi) [inline core/codenvy-hosted-workspace-activity/src/test/java/com/codenvy/activity/server/WorkspaceActivityServiceTest.java]: OfNonRunningWorkspace?

File: core/codenvy-hosted-workspace-activity/src/test/java/com/codenvy/activity/server/WorkspaceActivityServiceTest.java (hunk 1 above)
Comment (mkuznyetsov) [inline core/codenvy-hosted-workspace-activity/src/test/java/com/codenvy/activity/server/WorkspaceActivityServiceTest.java]: done

File: core/codenvy-hosted-workspace-activity/src/test/java/com/codenvy/activity/server/WorkspaceActivityServiceTest.java (hunk 2 above)
Comment (mkuznyetsov) [inline core/codenvy-hosted-workspace-activity/src/test/java/com/codenvy/activity/server/WorkspaceActivityServiceTest.java]: ok

File: core/codenvy-hosted-workspace-activity/src/test/java/com/codenvy/activity/server/WorkspaceActivityServiceTest.java (hunk 3 above)
Comment (mkuznyetsov) [inline core/codenvy-hosted-workspace-activity/src/test/java/com/codenvy/activity/server/WorkspaceActivityServiceTest.java]: renamed

File: core/codenvy-hosted-workspace-activity/src/test/java/com/codenvy/activity/server/WorkspaceActivityServiceTest.java (hunk 4 above)
Comment (mkuznyetsov) [inline core/codenvy-hosted-workspace-activity/src/test/java/com/codenvy/activity/server/WorkspaceActivityServiceTest.java]: ok

**Pull request merged**